src/ServiceControl.Audit.AcceptanceTests.RavenDB5/App.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <appSettings>
+    <add key="ServiceControl.Audit/ForwardAuditMessages" value="false" />
+    <add key="Raven/Esent/MaxVerPages" value="4096" />
```

Comment:
That doesn't apply to Voron, and I don't know if there is a Voron counterpart of MaxVerPages

```suggestion
```